plugins/inputs/win_perf_counters/README.md

```diff
@@ -19,7 +19,19 @@ as counters used when performance monitoring
  There are a lot of other good objects to monitor, if you know what to look for.
  This file is likely to be updated in the future with more examples for
  useful configurations for separate scenarios.
-
+ 
+ ### Schema
+ 
+ *Measurement name* is specified per performance object or `win_perf_counters` by default.
+ 
+ *Tags:*
+  - computer - computer name, if specified in the `Computers` parameter, otherwise omitted
+  - objectname - normalized name of the performance object
```

Comment:
Snake case this - `object_name`

Reply:
In this part of readme, I just added schema description, which was missing. 
`objectname` is legacy tag name. Even it is not according to best practises, I  think we cannot change it now.